keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -760,13 +1144,38 @@ as a way of doing things like:
   "HOLD" value.
 - Validate that only a controller responsible for a finalizer can remove it from
   the finalizers field.
+
+This could be supported by matching criteria, or via CEL expression access, or both.
  
  Concerns: "Is joe authorized to do this"? That only works for the objects joe
  creates, but not objects that get created on joe's behalf by a controller.
  Ditto for updates. I heard someone cite PSP as an example for why it's needed,
  but IMO that was an anti-pattern of PSP, and one that we explicitly decided to
  omit from PSA
 
+#### Default matching
```

Comment:
What is the use case behind default matching?

What is the behavior if a user defines match criteria:

* on the policy definition (not relevant for some of these alternatives)
* on the policy config object

Is the default matching overridden or evaluated in addition to the user-specified match criteria

Reply:
Primary use case is defining a least privilege policy, e.g. "Restrict maxReplicas to 5 unless there is a policy configuration that sets a different limit" or "Do not allow any of the following label keys to be used unless there is a policy configuration allowing them for the resource being admitted".

For these cases to work, the default matching would on apply if no policy configuration match.

There are cases where the desired effect doesn't require feature support and can just have a normal policy configuration that covers the "default case", e.g. If you have a global policy configuration for some upper limit to maxReplicas (say 1000), you could also have lower limits for scoped policy configurations without needing this "default configuration" feature.  Or, you can use namespace label selectors (if the namespace's environment label not in ['test', 'staging', 'prod']) to define what the default configuration should match.

The idea here is to generalize support for least privilege policy scenarios. Useful? Could be added later safely?

Reply:
I found a simpler way to incorporate this. See "Default Validations" in latest KEP version.